src/views/tickets/orders.php

```diff
@@ -44,7 +44,13 @@
 	<form method="post">
 
 	<?php tribe_tickets_get_template_part( 'tickets/orders-rsvp' ); ?>
-	<?php tribe_tickets_get_template_part( 'tickets/orders-tickets' ); ?>
+
+		<?php
+		/**
+		 * Adds ticket order form
+		 */
+		do_action( 'event_tickets_plus_orders_tickets' );
```

Comment:
Thank you! I knew the naming was off but couldn't come up with anything coherent ;p